applications/StructuralMechanicsApplication/python_scripts/kratos_main_structural.py

```diff
@@ -102,6 +106,9 @@
 time = start_time
 main_model_part.ProcessInfo[STEP] = 0
 
+if (parallel_type == "OpenMP") or (mpi.rank == 0):
+    Logger.PrintInfo("::[KSM Simulation]:: ", "Analysis -START- ")
```

Comment:
Just to say that, I plan to add a source to the logger so you can send the KSM Simulation by it.